Tuist/Templates/mvm/AppDelegate.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+final class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
Please fix this file for demo running too :)
```suggestion
@UIApplicationMain
final class AppDelegate: UIResponder, UIApplicationDelegate {
```